catalog/video/video-mpegts-transformer/src/main/java/org/codice/alliance/transformer/video/MpegTsInputTransformer.java

```diff
@@ -132,6 +144,110 @@ public Metacard transform(InputStream inputStream)
         return transform(inputStream, null);
     }
 
+    /**
+     *
+     * @param classification must be non-null
+     */
+    public void setSecurityClassificationCode1(String classification) {
+        notNull(classification, CLASSIFICATION_MUST_BE_NON_NULL);
+        klvProcessor.accept(new SetSecurityClassificationString((short) 1, classification));
+    }
+
+    /**
+     *
+     * @param classification must be non-null
+     */
+    public void setSecurityClassificationCode2(String classification) {
+        notNull(classification, CLASSIFICATION_MUST_BE_NON_NULL);
+        klvProcessor.accept(new SetSecurityClassificationString((short) 2, classification));
+    }
+
+    /**
+     *
+     * @param classification must be non-null
+     */
+    public void setSecurityClassificationCode3(String classification) {
+        notNull(classification, CLASSIFICATION_MUST_BE_NON_NULL);
+        klvProcessor.accept(new SetSecurityClassificationString((short) 3, classification));
+    }
+
+    /**
+     *
+     * @param classification must be non-null
+     */
+    public void setSecurityClassificationCode4(String classification) {
+        notNull(classification, CLASSIFICATION_MUST_BE_NON_NULL);
+        klvProcessor.accept(new SetSecurityClassificationString((short) 4, classification));
+    }
+
+    /**
+     *
+     * @param classification must be non-null
+     */
+    public void setSecurityClassificationCode5(String classification) {
+        notNull(classification, CLASSIFICATION_MUST_BE_NON_NULL);
+        klvProcessor.accept(new SetSecurityClassificationString((short) 5, classification));
+    }
+
+    /**
+     *
+     * @param classification must be non-null
+     */
+    public void setSecurityClassificationDefault(String classification) {
+        notNull(classification, CLASSIFICATION_MUST_BE_NON_NULL);
+        klvProcessor.accept(new KlvProcessor.Visitor() {
```

Comment:
Should we provide a `Visitor` base class with empty (no-op) implementations so that people can extend from it and only need to override the methods they care about?